include/TestMacros.h

```diff
@@ -299,7 +299,33 @@ namespace Mezzanine
                         AddTestResult( Mezzanine::Testing::TestData( (Name), (ExistingResult),                         \
                         __func__, __FILE__, __LINE__)) ;
             #endif
+        #endif
 
+        #ifndef TEST_WITHIN_RANGE
+            /// @def TEST_EQUAL
+            /// @brief The simplest test for comparing two values in a currently running UnitTestGroup.
+            /// @details  This captures test location meta data and should be considered the default way to record
+            /// equality tests.
```

Comment:
Equality tests?  With no qualifiers?  Then when do we use TEST_EQUAL?

Reply:
Copypasta mistake, will re-work.